editor/components/block-list/block.js

```diff
@@ -206,10 +206,10 @@ export class BlockListBlock extends Component {
 	 * @see https://developer.mozilla.org/en-US/docs/Web/Events/mouseenter
 	 */
 	maybeHover() {
-		const { isMultiSelected, isSelected } = this.props;
+		const { isPartOfMultiSelection, isSelected } = this.props;
 		const { isHovered } = this.state;
 
-		if ( isHovered || isMultiSelected || isSelected ||
```

Comment:
I feel like we can drop this.props.isMultiSelecting entirely and replace it with `isPartOfMultiSelection`. Is there a use-case where we need both?

Reply:
The isMultiSelecting condition is required to avoid the hover effect when dragging the mouse to multi-select. There is a moment when adding a new block to a multi-selection where the block is not yet multi-selected and we are with the mouse over it without thus the hover would appear. The isMultiSelecting flag may be true while isPartOfMultiSelection maybe false.

Reply:
Ah, you mean dropping isMultiSelected entirely. If that's case in function maybeHover we are already doing it and we don't use isMultiSelected. But for other functions e.g: related to the block focus we should continue to use isMultiSelected because they want to check if the specific block is selected and not if the block or one ancestor is selected.

Reply:
I took a look at the other tests and I believe `isPartOfMultiSelection` probably makes more sense there as well.

Reply:
Thank you for watching the use cases in detail. I applied your suggestion and I made some tests and the result looks great :)